fsd_utils/simple_utils/data_utils.py

```diff
@@ -0,0 +1,16 @@
+import requests
+from flask import current_app
+
+
+def get_remote_data_as_json(endpoint):
+
+    response = requests.get(endpoint)
+    if response.status_code == 200:
+        data = response.json()
+        return data
+    else:
+        current_app.logger.warn(
+            "GET remote data call was unsuccessful with status code:"
+            f" {response.status_code} body: {response.content}."
+        )
+        return None
```

Comment:
Personally not a fan of returning `None` since it might just get caught as a `TypeError` further up the stack

I'd probably recommend using [raise_for_status](https://requests.readthedocs.io/en/latest/api/#requests.Response.raise_for_status) & let any [exceptions](https://requests.readthedocs.io/en/latest/api/#exceptions) bubble up, but it looks in this case to need to return safely so maybe not applicable